config/yurt-controller/prometheus/monitor.yaml

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
maybe we don't need to consider to provide prometheus configuration for yurt-controller-manager. so how about remove this file to keep simplity?